src/Tribe/Customizer/Single_Event.php

```diff
@@ -43,7 +43,10 @@ public function get_css_template( $template ) {
 			';
 		}
 
-		if ( $customizer->has_option( $this->ID, 'post_title_color' ) ) {
+		if (
+			! $customizer->has_option( $this->ID, 'post_title_color_choice' )
```

Comment:
@Camwyn this will break the V1 customizer. If a user switches to V2, sets the `post_title_color_choice`, then switches back to V1, this will always have this option. Instead, return early above if V2.